app/_hub/kong-inc/vault-auth/index.md

```diff
@@ -33,37 +33,44 @@ params:
     - name: access_token_name
       required: true
       default: "`access_token`"
+      datatype: array of string elements
       description: |
         Describes an array of comma-separated parameter names where the plugin will look for an access token. The client must send the access token in one of those key names, and the plugin will try to read the credential from a header or the querystring parameter with the same name.<br>*note*: the key names may only contain [a-z], [A-Z], [0-9], [_] and [-].
     - name: vault.id
       required: true
       default:
       value_in_examples: "<UUID>"
+      datatype: foreign
```

Comment:
```suggestion
      datatype: foreign UUID
```

vs

```suggestion
      datatype: foreign ID
```

what do you prefer? or just `foreign` is fine?

what is the current convention used in the site across plugins? If there is one existing convention, I'd stick to that. If not, then we create a convention now.

`foreign` is an ID referencing an existing element in the database which is identified by its ID

Reply:
> what do you prefer? or just foreign is fine?

@murillopaula I'm not aware yet of any other plugin that has this so we are creating a convention now. I think foreign ID is better.

Reply:
okay, let's go with foreign ID then

Reply:
okay we can always change it later